examples/pub-sub-message-encryption/page.md

```diff
@@ -0,0 +1,120 @@
+# Sending and receiving encrypted messages with Pub/Sub
+
+Enable clients to send and receive encrypted messages in a channel.
+
+Messaging enables instant, scalable communication through publishing and subscribing to channels. Messages represent dynamic data payloads that clients publish and subscribe to, powering collaborative applications across realtime newsfeeds, auction applications or financial websites.
+
+Encryption is enabled by default in the Ably SDKs ensuring data is sent and received securely. Messages however are not encrypted and cannot be decrypted by Ably and are encrypted on a channel by channel basis.
```

Comment:
```suggestion
TLS (Transport Layer Security) is enabled by default in Ably SDKs to ensure data is sent and received securely. However, messages are not encrypted by default. Encryption can be configured on a channel-by-channel basis to make message payloads opaque, so that they can only be decrypted by other clients sharing a secret key.
```